api/nakadi-event-bus-api.yaml

```diff
@@ -1647,6 +1695,17 @@ definitions:
         type: string
         description: |
           Offset of the event being pointed to.
+      shift:
+        type: number
+        description: |
+          This number is a modifier for the offset. It moves the cursor forward or backwards by the amount provided.
+          For example, suppose a user wants to read events starting 100 positions before "001-000D-0000000000000009A8",
+          it's possible to specify shift with -100 and Nakadi will make the necessary calculations to move the cursor
+          relatively to the given offset.
```

Comment:
Should we indicate here what happens if the user provides a shift that's either too far in the past (before BEGIN), or too far in the future (not enough events have been published yet)?